Sources/Import/ClangImporter.swift

```diff
@@ -477,7 +477,9 @@ class ClangImporter: Pass {
       stringExpr.type = .pointer(type: .int8)
       return stringExpr
     case .number(let value, let raw):
-      return NumExpr(value: value, raw: raw, sourceRange: range)
+      let expr = NumExpr(value: value, raw: raw, sourceRange: range)
+      expr.type = .int64
```

Comment:
Nope, it won't. Hmm... I'm going to rewrite this in terms of `Cursor.evaluate()` sometime soon